### PR TITLE
First pass at contributing docs

### PR DESCRIPTION
**Pull request by mccurdyc**
## Proposed Change(s)
* First pass at contributing docs.
* Documents a major "gotcha" in updating the recordings for an existing resource.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,12 +175,21 @@ submitting a pull request with new or updated fixtures, we ask that contributors
...
 `FASTLY_TEST_SERVICE_ID` set to the same value used to run your tests.
 
+If you are **updating** an existing resource, in order to force go-vcr to re-record,
+rather than use the existing recorded values, you will have to removed the recorded
+values for that resource via `rm -r fastly/fixtures/<resource_name>`. **If you don't
+remove the existing recorded values, you will see the following error**:
+
+```
+Post "https://api.fastly.com/service/SERVICEID/version": Requested interaction not found
+```
```
Comment (mccurdyc) [inline README.md:185]: @phamann this is a major currently-undocumented "gotcha".

File: README.md [hunk 2]
```diff
@@ -177,19 +174,33 @@ submitting a pull request with new or updated fixtures, we ask that contributors
...
 
 # Re-run test suite with newly recorded fixtures
 unset FASTLY_TEST_SERVICE_ID FASTLY_API_KEY
 make test
 ```
 
+Contributing
+--------------------------
+
+Refer to [CONTRIBUTING.md](./CONTRIBUTING.md)
```
Comment (mccurdyc) [inline README.md:202]: @phamann applied the change here also.

File: Makefile [hunk 3]
```diff
@@ -49,9 +50,9 @@ test-race: ## Runs the test suite with the -race flag to identify race condition
 
 test-full: ## Runs the tests with VCR disabled (i.e., makes external calls).
 	@echo "==> Testing ${NAME} with VCR disabled"
-	@env \
-		VCR_DISABLE=1 \
-		go test -timeout=60s -parallel=20 ${GOPKGS} ${TESTARGS}
+	@VCR_DISABLE=1 \
```
Comment (mccurdyc) [inline Makefile:53]: This feels wrong. Especially since we specify the shell above.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Contributing
+
```
Comment (phamann) [inline CONTRIBUTING.md:2]: I quite like the additional context/paragraph you included in the CLI branch:
```suggestion

We're happy to receive feature requests and PRs. If your change is nontrivial,
please open an [issue](https://github.com/fastly/cli/issues/new) to discuss the
idea and implementation strategy before submitting a PR.

```

File: README.md (hunk 1 above)
Comment (phamann) [reply README.md:184]: Agreed, a good addition.

**Review by phamann — APPROVED** [review]
LGTM other than minor suggestion.

File: CONTRIBUTING.md (hunk 4 above)
Comment (mccurdyc) [reply CONTRIBUTING.md:2]: Fixed link to `go-fastly`.

File: README.md (hunk 1 above)
Comment (mccurdyc) [reply README.md:184]: Yeah, I know I've spent hours trying to debug this by diving into the backend, only to find out that I needed to remove the fixtures.

**Pull request merged**